app/controllers/users/emails_controller.rb

```diff
@@ -52,6 +53,7 @@ def confirm_delete
     end
 
     def delete
+      send_delete_email_notification
```

Comment:
Won't this send an email even if the email is not deleted?

Reply:
We probably want something like this after in the success block:

```ruby
(current_user.confirmed_email_addresses + [email_address]).each do |notif_email_address| do
  # ...
end
```

Reply:
Will fix. Yes, I wanted the email address being deleted to receive notification as well